server/routes/pubDocument.tsx

```diff
@@ -50,6 +52,7 @@ const renderPubDocument = (res, pubData, initialData) => {
 				title: pubData.title,
 				unlisted: !pubData.isRelease,
 			})}
+			bodyClassPrefix="content"
```

Comment:
Curious about the choice of this versus `pub`.

Reply:
yeah, I'm being too clever.